src/source/aws_cognito_login_screen_roku/tests/AWSLoginScreenVMTests.bs

```diff
@@ -51,7 +51,15 @@ function beforeEach()
       "input_placeholder_color": "#ff0000ff"
       "background_color": "#ff0000ff"
     }
+    "localizations":{}
```

Comment:
hope this didn't cause you grief.. in sdk4, I have a guard on localizations so we don't have to do this in every vm test.